source/ai-data-usage.txt

```diff
@@ -0,0 +1,35 @@
+.. vscode-ai-data-usage:
+
+=============================
+AI and Data Usage Information
+=============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+The MongoDB Github Copilot Participant is powered by Generative AI (Gen
+AI), and may give inaccurate responses. See our `Generative AI FAQ
+<https://dochub.mongodb.org/core/faq-ai-features>`__ for more
+information about Gen AI in MongoDB products.
+
+Third Party Providers
+---------------------
+
+The MongoDB Github Copilot Participant is powered by `Github Copilot <https://github.com/features/copilot>`__.
+
+How Your Data is Used
```

Comment:
I'd like us to make sure to run this answer by Product legal if we haven't yet. This contains what we normally include but because we get so many questions from customers it's just good to be sure we have covered our bases here.   We've been primarily working with Rachael McClure  - let me know if you need any assistance in getting a check here.  cc @GaurabAryal